src/java/org/apache/cassandra/service/paxos/CommitVerbHandler.java

```diff
@@ -42,6 +42,8 @@ public void doVerb(Message<Commit> message)
         RequestSensors sensors = new RequestSensors();
         Context context = Context.from(message.payload.update.metadata());
         sensors.registerSensor(context, Type.WRITE_BYTES);
```

Comment:
Not clear why we need to register this sensor here since we don't add anything to it. Test also passes fine without it